### PR TITLE
allow to configure reconcile_commit_every

### PR DESCRIPTION
**Pull request by lepistone**
this allows to configure if we want to commit the automatic reconcile every
N lines. It is left to zero by default to avoid possible problems with
tests that rely on a transaction.

### CODE REVIEW

File: account_advanced_reconcile/__init__.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
 ##############################################################################
 #
-#    Author: Guewen Baconnier
-#    Copyright 2012 Camptocamp SA
+#    Author: Guewen Baconnier, Leonardo Pistone
```
Comment (pedrobaeza) [inline account_advanced_reconcile/__init__.py]: I think you must put as contributor, not author.

**Comment by lepistone** [comment]
This continues the discussion started on https://github.com/OCA/bank-statement-reconcile/pull/12 with @guewen @lmignon @sebastienbeau @bwrsandman.

Thanks!

**Comment by lepistone** [comment]
@pedrobaeza I changed according to your remark. Thanks!

**Comment by guewen** [comment]
:+1:

**Comment by lmignon** [comment]
:+1:  
Thank you for this improvement to your initial PR!

**Comment by bwrsandman** [comment]
:+1:

**Pull request merged**